### PR TITLE
Stating fixing3

### PR DESCRIPTION
**Pull request by Bod9001**
### Changelog:
CL: [New]  health scanner tells you if Target is hungry
CL: [Improvement] Makes it so when you get ghosted it cancels flash effect
CL: [Improvement] devs spawner you can now search for objects that inherit from an object 
CL: [Balance] balances internal organ chemistry, to increase the speed of healing for vital organs
CL: [Fix] fixes Fire locks active blocking doors from being opened
CL: [Fix] Redirect, damage to crotch area, Bullets will no longer disappear into your groin doing no damage, now they wiill be redirected into your chest
CL: [Fix] fixes escape shuttle redocking at Central command and bugged escape shuttle ( en route to central command bug)

### CODE REVIEW

**Comment by corp-0** [comment]
> health scanner tells you if Target is hungry

what's the justification?

**Comment by MaxIsJoe** [comment]
> > health scanner tells you if Target is hungry
> 
> what's the justification?

Because some body parts can induce hunger individually. If you scan like a borg with a human limb; you'd probably want to be able to tell if their limbs are inducing hunger.

File: UnityProject/Assets/Scripts/Core/Camera/CameraEffectControlScript.cs [hunk 1]
```diff
@@ -120,6 +120,15 @@ private IEnumerator FlashEyesCoroutine(float flashTime)
 			lastFlashbangCoroutine = null;
 		}
 
+		public void Stop()
```
Comment (MaxIsJoe) [inline UnityProject/Assets/Scripts/Core/Camera/CameraEffectControlScript.cs:123]: Why flashes only?

There's like 5 other different effects that we have; and I'm pretty sure we already have a function somewhere that handles this.

**Review by MaxIsJoe — CHANGES_REQUESTED** [review]

**Comment by corp-0** [comment]
> > > health scanner tells you if Target is hungry
> > 
> > 
> > what's the justification?
> 
> Because some body parts can induce hunger individually. If you scan like a borg with a human limb; you'd probably want to be able to tell if their limbs are inducing hunger.

damn you hungry beards

**Comment by Bod9001** [comment]
The hunger system still summarises it to work out which icon to show

File: UnityProject/Assets/Scripts/Core/Camera/CameraEffectControlScript.cs (hunk 1 above)
Comment (Bod9001) [reply UnityProject/Assets/Scripts/Core/Camera/CameraEffectControlScript.cs:123]: what other effects are there?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Core/Camera/CameraEffectControlScript.cs:123]: Take a look at the camera prefab

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Core/Camera/CameraEffectControlScript.cs:123]: all the others are controlled by body synchronise stuff

**Comment by MaxIsJoe** [comment]
pain

**Pull request merged**